src/packages/tests/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -88,14 +80,19 @@ type Database<Client> = {
    * @remarks This is used as the default provider name for the Prisma schema datasource block.
    */
   name: string
+  /** 
+   * Supply the enabled preview features for the prisma client. 
+   * 
+   */
+  previewFeatures?: string[] 
   /**
    * Create a client connection to the database.
    */
   connect: (ctx: Context) => MaybePromise<Client>
   /**
-   * Execute SQL against the database.
+   * Execute SQL for a scenario against the database. 
    */
-  send: (db: Client, sql: string) => MaybePromise<any>
+  up: (db: Client, sqlScenario: string, ctx: Context) => MaybePromise<any>
```

Comment:
send() was designed as a no side-effects sql execution method, but can't be used that way anymore, which the mssql case shows. @madebysid and I agreed that re-naming to `up` and passing in the context makes more sense

Reply:
AKA: There's no `send` anymore, and we're doing all scenario setup inside `up`.

In case of MSSQL, we have to pass in DB setup & scenario setup in separate calls (instead of concatenating them as before), so all of this is done in `up` now, rather than branching within the `integrationTest` utility